server/utils/nunjucksSetup.ts

```diff
@@ -45,4 +54,16 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     const array = fullName.split(' ')
     return `${array[0][0]}. ${array.reverse()[0]}`
   })
+
+  njkEnv.addFilter('findError', (array: Error[], formFieldId: string) => {
+    const item = array?.find(error => error.href === `#${formFieldId}`)
+    if (item) {
```

Comment:
could replace this if condition block into a one liner
`return item ? { text: item.message } : null`

Reply:
👍